style.css

```diff
@@ -1 +1,179 @@
 /*  Add your styling here */
+@media(max-width: 1024px){
+     
+  .upper-navigation{
+    justify-content: center;
+    font-size: 1.2rem;
+    margin-top: 25px; 
+    -webkit-font-smoothing: antialiased;
+  }
+  
+  .upper-navigation li {
+    display: inline-block;
+    margin-left: 75px;
+  }
+
+
+    
+    
+  
+}
+
+body{
+    background-color: pink;
+}
+
+.upper-navigation {
+    justify-content: space-around;
```

Comment:
in order for this to have any affect, you will need to make use of `display: flex` - have a look at last weeks lesson to look over flexbox and how this is used to remind yourself